src/app/page.tsx

```diff
@@ -29,8 +37,11 @@ export default function Home() {
         </HeroSection>
 
         <OurClients />
+        <QueryProvider />
```

Comment:
not correct name for component, this component getting and display list with posts and filters

Reply:
fixed